CONTRIBUTING.md

```diff
@@ -131,6 +131,62 @@ There are three PR templates. The [default template](./.github/PULL_REQUEST_TEMP
 
 * `template=docs.md`
 
+### Pull Request Accountability
+
+#### PR Owner
+
+The PR owner is responsible for ensuring that the PR is ready for review and merging after reviews are delivered. This includes:
+
+* Ensuring that the PR is up to date with the latest changes in the main branch.
+* Ensuring that the PR passes all checks.
+* Ensuring that the PR has a clear description of the changes.
+* Ensuring that the PR has a clear description of the testing strategy.
+* Ensuring that the PR has a clear description of the impact of the changes.
+* Ensuring that the PR has a clear description of the risks associated with the changes.
+* Ensuring that the PR has a clear description of the next steps.
+* Ensuring that the PR has a clear description of the dependencies.
+
+The pull request owner is responsible for assigning reviewers on the team, responding to feedback, and ensuring that the PR is merged in a timely manner. If a PR is reviewed, but an approval is not given by the reviewer the pull request owner is responsible for addressing the feedback, ensuring that the PR is ready for review again and notifying the reviewers that the PR is ready for review. 
+
+Once approvals have been given by the reviewer(s) it is the responsibility of the pull request owner to merge the PR.
```

Comment:
Should we add that when the PR is from an external contributors, the merging will be done by one of the reviewers?

Reply:
yup yup, adding external contributor section 😄